internal/testing/factories/factories.go

```diff
@@ -0,0 +1,31 @@
+package factories
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"fmt"
+	"sync/atomic"
+	"testing"
+
+	"golang.org/x/crypto/ssh"
+)
+
+var nameSequence uint32 = 1
```

Comment:
Super nitpick - 32bit numbers are slower on 64bit systems while they take the same amount of memory. But no biggie 32bit ought to be enough for everybody (TM). ;)

Reply:
well, most HW we run on will be probably 64, so probably better, lets go with 64 ^_^

Reply:
ALL of the hardware. My God do not play with fire! :)